phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1040,11 +1038,9 @@ public function format_methodparam($open, $name, $attrs) {
                     $content .= " (";
                 }
                 if (isset($attrs[Reader::XMLNS_DOCBOOK]["choice"]) && $attrs[Reader::XMLNS_DOCBOOK]["choice"] == "opt") {
-                    $this->params["opt"]++;
-                    $content .= "[";
+                    $this->params["opt"] = true;
                 } else if($this->params["opt"]) {
-                    $content .= str_repeat("]", $this->params["opt"]);
-                    $this->params["opt"] = 0;
+                    $this->params["opt"] = false;
```

Comment:
Just an `else` would be clearer imho. We can unconditionally overwrite it, rather than only if it is not already false.